### PR TITLE
translate Kubernetes 1.24: Stargazer into Chinese

### PR DESCRIPTION
**Pull request by sarazqy**
translate Kubernetes 1.24: Stargazer into Chinese

### CODE REVIEW

**Comment by sarazqy** [comment]
fix https://github.com/kubernetes/website/issues/34192

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+<!--
+A Service `ClusterIP` can be assigned:
+
+* dynamically, which means the cluster will automatically pick a free IP within the configured Service IP range.
+* statically, which means the user will set one IP within the configured Service IP range.
+
+Service `ClusterIP` are unique; hence, trying to create a Service with a `ClusterIP` that has already been allocated will return an error.
+-->
+
+可以分配服务 `ClusterIP` ：
```
Comment (sftim) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:204]: I'd expect to see “Service” here; the API kind is Service.

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+* [Add Suspend Field to Jobs API](https://github.com/kubernetes/enhancements/issues/2232): Add a suspend field to the Jobs API to allow orchestrators to create jobs with more control over when pods are created.
+* [Pod Affinity NamespaceSelector](https://github.com/kubernetes/enhancements/issues/2249): Add a `namespaceSelector` field for to pod affinity/anti-affinity spec.
+* [Leader Migration for Controller Managers](https://github.com/kubernetes/enhancements/issues/2436): kube-controller-manager and cloud-controller-manager can apply new controller-to-controller-manager assignment in HA control plane without downtime.
+* [CSR Duration](https://github.com/kubernetes/enhancements/issues/2784): Extend the CertificateSigningRequest API with a mechanism to allow clients to request a specific duration for the issued certificate.
+-->
+
+* [容器存储接口（CSI）卷扩展](https://github.com/kubernetes/enhancements/issues/284)
+* [Pod Overhead](https://github.com/kubernetes/enhancements/issues/688): 核算与 pod 沙盒绑定的资源，但不包括特定的容器。
+* [向 PriorityClasses 添加非抢占选项](https://github.com/kubernetes/enhancements/issues/902)
```
Comment (sftim) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:308]:
```suggestion
* [向 PriorityClass 添加非抢占选项](https://github.com/kubernetes/enhancements/issues/902)
```
?

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/).
+-->
+
+## 主要议题
+
+### 从 kubelet 中删除 Dockershim
+
+在 v1.20 版本中被废弃后，dockershim 组件已从 Kubernetes v1.24 版本的 kubelet 中移除。
+从v1.24开始，如果你依赖Docker Engine 作为容器运行时，则需要使用其他[受支持的运行时](/docs/setup/production-environment/container-runtimes/)之一（如 containerd 或 CRI-O）或使用 CRI dockerd。
+有关确保群集已准备好进行此删除的更多信息，请参阅[本指南](/blog/2022/03/31/ready-for-dockershim-removal/)。
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:52]:
```suggestion
有关确保群集已准备好进行此删除的更多信息，请参阅[本指南](/zh-cn/blog/2022/03/31/ready-for-dockershim-removal/)。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 4]
```diff
@@ -0,0 +1,550 @@
...
+Release artifacts are [signed](https://github.com/kubernetes/enhancements/issues/3031) using [cosign](https://github.com/sigstore/cosign)
+signatures,
+and there is experimental support for [verifying image signatures](/docs/tasks/administer-cluster/verify-signed-images/).
+Signing and verification of release artifacts is part of [increasing software supply chain security for the Kubernetes release process](https://github.com/kubernetes/enhancements/issues/3027).
+-->
+
+### 签署发布工件
+
+发布工件使用 [cosign](https://github.com/sigstore/cosign) 签名进行[签名](https://github.com/kubernetes/enhancements/issues/3031)，
+并且有[验证图像签名](/docs/tasks/administer-cluster/verify-signed-images/)的实验性支持。
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:78]:
```suggestion
并且有[验证镜像签名](/zh-cn/docs/tasks/administer-cluster/verify-signed-images/)的实验性支持。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 5]
```diff
@@ -0,0 +1,550 @@
...
+发布工件的签名和验证是[提高 Kubernetes 发布过程的软件供应链安全性](https://github.com/kubernetes/enhancements/issues/3027)
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:79]:
```suggestion
发布工件的签名和验证是[提高 Kubernetes 发布过程的软件供应链安全性](https://github.com/kubernetes/enhancements/issues/3027)的一部分。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 6]
```diff
@@ -0,0 +1,550 @@
...
+的一部分。
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:80]:
```suggestion
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 7]
```diff
@@ -0,0 +1,550 @@
...
+and enhances scheduling of pods that use CSI volumes with late binding.
+
+[Volume expansion](https://github.com/kubernetes/enhancements/issues/284) adds support 
+for resizing existing persistent volumes. 
+-->
+
+### 存储容量和卷扩展普遍可用
+
+[存储容量跟踪](https://github.com/kubernetes/enhancements/issues/1472)支持通过
+[CSIStorageCapacity 对象](/docs/concepts/storage/storage-capacity/#api)公开当前可用的存储容量，
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:106]:
```suggestion
[CSIStorageCapacity 对象](/zh-cn/docs/concepts/storage/storage-capacity/#api)公开当前可用的存储容量，
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 8]
```diff
@@ -0,0 +1,550 @@
...
+Originally released as Alpha in Kubernetes 1.20, the kubelet's support for
+[image credential providers](/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
+has now graduated to Beta.
+This allows the kubelet to dynamically retrieve credentials for a container image registry
+using exec plugins rather than storing credentials on the node's filesystem.
+-->
+
+### Kubelet 凭证提供者毕业至 Beta
+
+kubelet 最初在 Kubernetes 1.20 中作为 Alpha 发布，现在它对[图像凭证提供者](/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
```
Comment (zhangxyjlu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:166]:
```suggestion
kubelet 最初在 Kubernetes 1.20 中作为 Alpha 发布，现在它对[镜像凭据提供者](/zh-cn/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md (hunk 6 above)
Comment (sarazqy) [reply content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:80]: 段落超长换行；

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md (hunk 5 above)
Comment (sarazqy) [reply content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:79]: 段落超长换行；

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 9]
```diff
@@ -0,0 +1,549 @@
...
+This release consists of 46 enhancements: fourteen enhancements have graduated to stable,
+fifteen enhancements are moving to beta, and thirteen enhancements are entering alpha.
+Also, two features have been deprecated, and two features have been removed.
+-->
+
+**作者**: [Kubernetes 1.24 发布团队](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md)
+
+我们很高兴地宣布 Kubernetes 1.24 的发布，这是 2022 年的第一个版本！
+
+这个版本包括 46 个增强功能：14 个增强功能已经升级到稳定版，15 个增强功能正在进入测试版，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:31]:
```suggestion
这个版本包括 46 个增强功能：14 个增强功能已经升级到稳定版，15 个增强功能正在进入 Beta 版，
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 10]
```diff
@@ -0,0 +1,549 @@
...
+13 个增强功能正在进入 alpha 阶段。另外，有两个功能被废弃了，还有两个功能被删除了。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:32]:
```suggestion
13 个增强功能正在进入 Alpha 阶段。另外，有两个功能被废弃了，还有两个功能被删除了。     
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 11]
```diff
@@ -0,0 +1,549 @@
...
+or use cri-dockerd if you are relying on Docker Engine as your container runtime.
+For more information about ensuring your cluster is ready for this removal, please
+see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/).
+-->
+
+## 主要议题
+
+### 从 kubelet 中删除 Dockershim
+
+在 v1.20 版本中被废弃后，dockershim 组件已从 Kubernetes v1.24 版本的 kubelet 中移除。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:50]:
```suggestion
在 v1.20 版本中被废弃后，dockershim 组件已被从 Kubernetes v1.24 版本的 kubelet 中移除。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 12]
```diff
@@ -0,0 +1,549 @@
...
+从v1.24开始，如果你依赖Docker Engine 作为容器运行时，则需要使用其他[受支持的运行时](/docs/setup/production-environment/container-runtimes/)之一（如 containerd 或 CRI-O）或使用 CRI dockerd。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:51]:
```suggestion
从 v1.24 开始，如果你依赖 Docker Engine 作为容器运行时，
则需要使用其他[受支持的运行时](/zh-cn/docs/setup/production-environment/container-runtimes/)之一
（如 containerd 或 CRI-O）或使用 cri-dockerd。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 13]
```diff
@@ -0,0 +1,549 @@
...
+### Beta APIs Off by Default
+
+[New beta APIs will not be enabled in clusters by default](https://github.com/kubernetes/enhancements/issues/3136).
+Existing beta APIs and new versions of existing beta APIs will continue to be enabled by default.
+-->
+
+### 默认情况下关闭 Beta API
+
+[新的 beta API 默认不会在集群中启用](https://github.com/kubernetes/enhancements/issues/3136)。
+默认情况下，现有 beta API 和现有 beta API 的新版本将继续启用。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:64]:
```suggestion
默认情况下，现有 Beta API 和及其更新版本将继续被启用。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 14]
```diff
@@ -0,0 +1,549 @@
...
+Work is underway to [migrate the internals of in-tree storage plugins](https://github.com/kubernetes/enhancements/issues/625) to call out to CSI Plugins
+while maintaining the original API.
+The [Azure Disk](https://github.com/kubernetes/enhancements/issues/1490)
+and [OpenStack Cinder](https://github.com/kubernetes/enhancements/issues/1489) plugins
+have both been migrated.
+-->
+
+### 存储插件迁移
+
+目前正在进行[树内存储插件的内部迁移](https://github.com/kubernetes/enhancements/issues/625)工作，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:134]:
```suggestion
目前正在进行[迁移树内存储插件的内部组件](https://github.com/kubernetes/enhancements/issues/625)工作，
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 15]
```diff
@@ -0,0 +1,549 @@
...
+has entered beta and is available by default. You can now [configure startup, liveness, and readiness probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) for your gRPC app
+natively within Kubernetes without exposing an HTTP endpoint or
+using an extra executable.
+-->
+
+
+### gRPC 探针升级到 Beta
+
+在 Kubernetes 1.24 中，[gRPC 探测功能](https://github.com/kubernetes/enhancements/issues/2727)
+已进入测试版，默认可用。现在，你可以在 Kubernetes 中为你的 gRPC 应用程序原生地[配置启动、活跃度和就绪性探测](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes)，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:151]:
```suggestion
已进入测试版，默认可用。现在，你可以在 Kubernetes 中为你的 gRPC
应用原生地[配置启动、活跃度和就绪性探测](/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes)，
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 16]
```diff
@@ -0,0 +1,549 @@
...
+Originally released as Alpha in Kubernetes 1.20, the kubelet's support for
+[image credential providers](/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
+has now graduated to Beta.
+This allows the kubelet to dynamically retrieve credentials for a container image registry
+using exec plugins rather than storing credentials on the node's filesystem.
+-->
+
+### Kubelet 凭证提供者毕业至 Beta
+
+kubelet 最初在 Kubernetes 1.20 中作为 Alpha 发布，现在它对[图像凭证提供者](/zh-cn/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:166]:
```suggestion
kubelet 最初在 Kubernetes 1.20 中作为 Alpha 发布，现在它对[镜像凭证提供者](/zh-cn/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 17]
```diff
@@ -0,0 +1,549 @@
...
+<!--
+A Service `ClusterIP` can be assigned:
+
+* dynamically, which means the cluster will automatically pick a free IP within the configured Service IP range.
+* statically, which means the user will set one IP within the configured Service IP range.
+
+Service `ClusterIP` are unique; hence, trying to create a Service with a `ClusterIP` that has already been allocated will return an error.
+-->
+
+可以分配 `ClusterIP` 服务：
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:204]:
```suggestion
服务的 `ClusterIP` 可以按以下两种方式分配：
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 18]
```diff
@@ -0,0 +1,549 @@
...
+* [Indexed Job](https://github.com/kubernetes/enhancements/issues/2214): Add a completion index to Pods of Jobs with a fixed completion count.
+* [Add Suspend Field to Jobs API](https://github.com/kubernetes/enhancements/issues/2232): Add a suspend field to the Jobs API to allow orchestrators to create jobs with more control over when pods are created.
+* [Pod Affinity NamespaceSelector](https://github.com/kubernetes/enhancements/issues/2249): Add a `namespaceSelector` field for to pod affinity/anti-affinity spec.
+* [Leader Migration for Controller Managers](https://github.com/kubernetes/enhancements/issues/2436): kube-controller-manager and cloud-controller-manager can apply new controller-to-controller-manager assignment in HA control plane without downtime.
+* [CSR Duration](https://github.com/kubernetes/enhancements/issues/2784): Extend the CertificateSigningRequest API with a mechanism to allow clients to request a specific duration for the issued certificate.
+-->
+
+* [容器存储接口（CSI）卷扩展](https://github.com/kubernetes/enhancements/issues/284)
+* [Pod Overhead](https://github.com/kubernetes/enhancements/issues/688): 核算与 pod 沙盒绑定的资源，但不包括特定的容器。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:307]:
```suggestion
* [Pod 开销](https://github.com/kubernetes/enhancements/issues/688)：核算与 Pod 沙箱绑定的资源，但不包括特定的容器。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 19]
```diff
@@ -0,0 +1,549 @@
...
+
+<!--
+### Release Notes
+
+Check out the full details of the Kubernetes 1.24 release in our [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+-->
+
+### 发行说明
+
+在我们的 [发行说明](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md) 中查看 Kubernetes 1.24 版本的完整详细信息。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:348]:
```suggestion
在我们的[发行说明](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md) 中查看 Kubernetes 1.24 版本的完整详细信息。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 20]
```diff
@@ -0,0 +1,549 @@
...
+To get started with Kubernetes, check out these [interactive tutorials](/docs/tutorials/) or run local
+Kubernetes clusters using containers as “nodes”, with [kind](https://kind.sigs.k8s.io/).
+You can also easily install 1.24 using [kubeadm](/docs/setup/independent/create-cluster-kubeadm/).
+-->
+
+### 可用性
+
+Kubernetes 1.24 可在 [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.0) 上下载。
+要开始使用 Kubernetes，请查看这些[交互式教程](/docs/tutorials/)或在本地运行。
+Kubernetes 集群使用容器作为“节点”，有[种类](https://kind.sigs.k8s.io/)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:363]:
```suggestion
使用[kind](https://kind.sigs.k8s.io/)，可以将容器作为 Kubernetes 集群的“节点”。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 21]
```diff
@@ -0,0 +1,549 @@
...
+你还可以使用 [kubeadm](/docs/setup/independent/create-cluster-kubeadm/) 轻松安装 1.24。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:364]:
```suggestion
你还可以使用 [kubeadm](/zh-cn/docs/setup/independent/create-cluster-kubeadm/) 轻松安装 1.24。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 22]
```diff
@@ -0,0 +1,549 @@
...
+
+参与 Kubernetes 的最简单方法是加入符合你兴趣的众多 [特别兴趣组](https://github.com/kubernetes/community/blob/master/sig-list.md)(SIG) 之一。
+你有什么想向 Kubernetes 社区广播的内容吗？在我们的每周的[社区会议](https://github.com/kubernetes/community/tree/master/communication)
+上分享你的声音，并通过以下渠道：
+
+* 在 [Kubernetes Contributors](https://www.kubernetes.dev/) 网站上了解有关为 Kubernetes 做出贡献的更多信息
+* 在 Twitter 上关注我们 [@Kubernetesio](https://twitter.com/kubernetesio) 以获取最新更新
+* 加入社区讨论 [Discuss](https://discuss.kubernetes.io/)
+* 加入 [Slack](http://slack.k8s.io/) 社区
+* 在[服务器故障](https://serverfault.com/questions/tagged/kubernetes)上发布问题（或回答问题）。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:546]:
```suggestion
* 在 [Server Fault](https://serverfault.com/questions/tagged/kubernetes) 上发布问题（或回答问题）。
```

**Comment by sarazqy** [comment]
thanks to review again @tengqm

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 23]
```diff
@@ -0,0 +1,552 @@
...
+
+* [容器存储接口（CSI）卷扩展](https://github.com/kubernetes/enhancements/issues/284)
+* [Pod 开销](https://github.com/kubernetes/enhancements/issues/688): 核算与 Pod 沙箱绑定的资源，但不包括特定的容器。
+* [向 PriorityClass 添加非抢占选项](https://github.com/kubernetes/enhancements/issues/902)
+* [存储容量跟踪](https://github.com/kubernetes/enhancements/issues/1472) 
+* [OpenStack Cinder In-Tree 到 CSI 驱动程序迁移](https://github.com/kubernetes/enhancements/issues/1489)
+* [Azure 磁盘树到 CSI 驱动程序迁移](https://github.com/kubernetes/enhancements/issues/1490)
+* [高效的监视恢复](https://github.com/kubernetes/enhancements/issues/1904): kube-apiserver 重新启动后，可以高效地恢复监视。 
+* [Service Type=LoadBalancer 类字段](https://github.com/kubernetes/enhancements/issues/1959):
+引入新的服务注解 `service.kubernetes.io/load-balancer-class` ，允许在同一个集群中实现多个 `type: LoadBalancer` 服务在同一集群中的多个实现。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:317]:
```suggestion
* [Service Type=LoadBalancer 类字段](https://github.com/kubernetes/enhancements/issues/1959)：引入新的服务注解 `service.kubernetes.io/load-balancer-class` ，允许在同一个集群中实现多个 `type: LoadBalancer` 服务在同一集群中的多个实现。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 24]
```diff
@@ -0,0 +1,552 @@
...
+* [指数化的作业](https://github.com/kubernetes/enhancements/issues/2214): 为有固定完成数的作业的 Pod 添加完成指数。
+* [在 Jobs API 中增加 Suspend 字段](https://github.com/kubernetes/enhancements/issues/2232): 
+在 Jobs API 中增加一个 suspend 字段，允许协调者在创建作业时对 pod 的创建有更多控制。
+* [Pod Affinity NamespaceSelector](https://github.com/kubernetes/enhancements/issues/2249): 
+为 pod affinity/anti-affinity 规范添加一个 `namespaceSelector` 字段。
+* [控制器管理器的领导者迁移](https://github.com/kubernetes/enhancements/issues/2436):
+kube-controller-manager 和 cloud-controller-manager 可以在 HA 控制平面中应用新的控制器到控制器管理器分配，而无需停机。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:324]:
```suggestion
* [在 Job API 中增加 Suspend 字段](https://github.com/kubernetes/enhancements/issues/2232)：在 Job API 中增加一个 suspend 字段，允许协调者在创建作业时对 pod 的创建有更多控制。
* [Pod Affinity NamespaceSelector](https://github.com/kubernetes/enhancements/issues/2249)：为 Pod affinity/anti-affinity 规范添加一个 `namespaceSelector` 字段。
* [控制器管理器的领导者迁移](https://github.com/kubernetes/enhancements/issues/2436)： kube-controller-manager 和 cloud-controller-manager 可以在 HA 控制平面中应用新的控制器到控制器管理器分配，而无需停机。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 25]
```diff
@@ -0,0 +1,552 @@
...
+Kubernetes 1.24 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.0).
+To get started with Kubernetes, check out these [interactive tutorials](/docs/tutorials/) or run local
+Kubernetes clusters using containers as “nodes”, with [kind](https://kind.sigs.k8s.io/).
+You can also easily install 1.24 using [kubeadm](/docs/setup/independent/create-cluster-kubeadm/).
+-->
+
+### 可用性
+
+Kubernetes 1.24 可在 [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.0) 上下载。
+要开始使用 Kubernetes，请查看这些[交互式教程](/docs/tutorials/)或在本地运行。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:365]:
```suggestion
要开始使用 Kubernetes，请查看这些[交互式教程](/zh-cn/docs/tutorials/)或在本地运行。
```

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 26]
```diff
@@ -0,0 +1,552 @@
...
+### 生态系统更新
+
+* KubeCon + CloudNativeCon Europe 2022 将于 2022 年 5 月 16 日至 20 日在西班牙巴伦西亚举行！
+你可以在 [活动网站](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/)
+上找到有关会议和注册的更多信息。
+* 在 [2021 Cloud Native Survey](https://www.cncf.io/announcements/2022/02/10/cncf-sees-record-kubernetes-and-container-adoption-in-2021-cloud-native-survey/)
+中，CNCF 看到了创纪录的 Kubernetes 和容器采用。看看[调查结果](https://www.cncf.io/reports/cncf-annual-survey-2021/)。
+* [Linux 基金会](https://www.linuxfoundation.org/)和[云原生计算基金会](https://www.cncf.io/) (CNCF)
+宣布推出新的 [云原生开发者训练营](https://training.linuxfoundation.org/training/cloudnativedev-bootcamp/?utm_source=lftraining&utm_medium=pr&utm_campaign=clouddevbc0322)
+为参与者提供设计、构建和部署云原生应用程序的知识和技能。查看[公告](https://www.cncf.io/announcements/2022/03/15/new-cloud-native-developer-bootcamp-provides-a-clear-path-to-cloud-native-careers/)以了解更多信息。
```
[the remark this answers is not in the document]
Comment (ydFu) [reply content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:482]: Cannot be segmented.

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md (hunk 24 above)
Comment (ydFu) [reply content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:324]: Cannot be segmented.

File: content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md (hunk 23 above)
Comment (ydFu) [reply content/zh-cn/blog/_posts/2022-05-03-kubernetes-release-1.24.md:317]: Cannot be segmented.

**Comment by windsonsea** [comment]
The preview is good.

/lgtm

**Pull request merged**